src/server/pkg/deploy/assets/assets.go

```diff
@@ -506,6 +508,105 @@ func InitJob(version string) *extensions.Job {
 	}
 }
 
+func RegistryRc() *api.ReplicationController {
```

Comment:
Just to aid my own understanding, are this and etc/kube/registry-rc.yaml the same? If so, why define them both?